src/messageComposer/middleware/textComposer/textMiddlewareUtils.ts

```diff
@@ -134,3 +143,44 @@ export const getTokenizedSuggestionDisplayName = ({
       : [displayName],
   },
 });
+
+/**
+ * Utility function to check if a given input text matches another text.
+ * @param input
+ * @param textToBeMatchedWith
+ * @returns
+ */
+export const startsWithTextAndSpace = (
```

Comment:
I think we may not need this function. It is used in a single place and it actually removes white space that was kept previously due to allowing white space in getTriggerWithToken

Reply:
Well, this function just checks if the `query` starts with the command name or not, and we are obviously allowing at least one or more spaces so that only then it would return true for a match, i.e. `/unban ` will be a matched command and `/unban` would not for the onChange.